OpenImis.DB.SqlServer/TblPolicy.cs

```diff
@@ -31,6 +31,7 @@ public TblPolicy()
         public byte[] RowId { get; set; }
         public bool? IsOffline { get; set; }
         public Guid PolicyUUID { get; set; }
+        public bool SelfRenewed { get; set; }
```

Comment:
Was this field added in DB scripts?

Reply:
yes it is. I confirmed in the develop branch. It is merged.